py_zipkin/encoding/_encoders.py

```diff
@@ -21,6 +22,8 @@ def get_encoder(encoding):
         return _V1JSONEncoder()
     if encoding == Encoding.V2_JSON:
         return _V2JSONEncoder()
+    if encoding == Encoding.V2_PROTOBUF:
```

Comment:
not necessarily important to you, but we call this PROTO3 to differentiate between the format. There was no V1 encoding.. anyway you can decide what you think is right https://github.com/openzipkin/zipkin/blob/master/zipkin/src/main/java/zipkin2/codec/SpanBytesEncoder.java#L110